docs/en/guides/docs_contributing.rst

```diff
@@ -49,11 +49,11 @@ Before starting your collaboration, you need to get the documentation source cod
 Requirements
 ************
 
-To properly work with the documentation, you need to install some packages in your system.
+To properly work with the documentation, you need to install some packages in your system. Note that depending on
```

Comment:
I would recommend changing to: `To build the documentation properly, you need to...`